Client/Frontend/Browser/TopTabsViews.swift

```diff
@@ -4,6 +4,17 @@
 
 import Foundation
 
+class Seperator: UICollectionReusableView {
```

Comment:
Nit: I would make this naming more specific since it's exposed to all the source files in the package. Maybe TopTabSeparator? Also just noticed that all instances of `seperator`->`separator`